### ISSUE TITLE
Reporting Django1.10 deprecation warning

### ISSUE THREAD

**Issue by SalahAdDin** [wagtail/wagtail#2116]
Report this warning:

```
/run/media/salahaddin/Data/Proyectos/Trabajo/ccct/lib/python3.5/site-packages/wagtail/wagtailadmin/edit_handlers.py:529: RemovedInDjango110Warning: Usage of field.related has been deprecated. Use field.remote_field instead.
  related_model = get_related_parent_model(field.related)
```

**Comment by kaedroho** [comment]
Thanks! I assume you see this on a Django 1.9 project? Is this with default logging configuration?

**Comment by SalahAdDin** [comment]
Yes, I'm using Django 1.9 and wagtail 1.3.1. And is the terminal login in develop environment.

**Comment by gasman** [comment]
Apparently this never got closed, but Django 1.10 support has come and gone, so closing this as outdated.
